Objects/Converters/ConverterRevit/ConverterRevitShared/Partial Classes/ConvertDuct.cs

```diff
@@ -87,9 +91,9 @@ public ApplicationObject DuctToNative(BuiltElements.Duct speckleDuct)
         TrySetParam(duct, BuiltInParameter.RBS_VELOCITY, speckleRevitDuct.velocity, speckleRevitDuct.units);
 
         SetInstanceParameters(duct, speckleRevitDuct);
-      }
 
-      CreateSystemConnections(speckleRevitDuct.Connectors, duct, receivedObjectsCache);
+        CreateSystemConnections(speckleRevitDuct.Connectors, duct, receivedObjectsCache);
```

Comment:
This is the actual bug fix: for generic mep elements, `speckleRevitDuct` is null